.github/ISSUE_TEMPLATE/sip.md

```diff
@@ -1,10 +1,9 @@
 ---
 name: SIP
-about: Superset Improvement Proposal (See SIP-0: https://github.com/apache/superset/issues/5602)
```

Comment:
Might wanna get rid of that last parenthesis as well

Reply:
@rusackas could you be more specific which parenthesis is wrong?